lms/templates/index.html

```diff
@@ -61,30 +61,12 @@
     </div>
 
   </header>
-  <section class="courses-container">
-    <section class="highlighted-courses">
+  <%
+    # allow for microsite override of the courses list
+    courses_list = microsite.get_template_path('courses_list.html')
```

Comment:
This is fine for now, however just FYI in the next sprint we will be moving all references to 'microsite' behind the comprehensive theming abstraction